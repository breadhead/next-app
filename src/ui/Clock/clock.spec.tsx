import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

import { Clock } from './index';

describe('clock ui', () => {
  test('Button renders correctly', () => {
    const output = shallow(<Clock lastUpdate={12} />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
