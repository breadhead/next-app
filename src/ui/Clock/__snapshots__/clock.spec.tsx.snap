// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clock ui Button renders correctly 1`] = `
<div
  className=""
>
  00:00:00
</div>
`;
